ulmo/ssl/train_util.py

```diff
@@ -392,6 +379,8 @@ def train_model(train_loader, model, criterion, optimizer, epoch, opt, cuda_use=
 
     end = time.time()
     for idx, images in enumerate(train_loader):
+        if idx > 5:
```

Comment:
should this be removed?

Reply:
Yes, right. It is for the test of the code. I will fix that now and sorry about that.